mods/ra/weapons/smallcaliber.yaml

```diff
@@ -189,134 +191,75 @@ Vulcan:
 		Delay: 10
 
 ChainGun:
+	Inherits: ^HeavyMG
 	ReloadDelay: 10
 	Range: 5c0
 	MinRange: 0c768
-	Report: gun13.aud
 	Projectile: Bullet
-		Speed: 1c682
 		Blockable: false
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
-		Damage: 30
 		Versus:
 			None: 120
-			Wood: 50
-			Light: 60
-			Heavy: 25
-			Concrete: 25
-		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
-	Warhead@2Eff: CreateEffect
-		Explosions: piffs
-		InvalidImpactTypes: Water
-	Warhead@3EffWater: CreateEffect
-		Explosions: water_piffs
-		ValidImpactTypes: Water
 
 ChainGun.Yak:
+	Inherits: ^HeavyMG
 	ReloadDelay: 3
 	Range: 5c0
 	MinRange: 3c0
-	Report: gun13.aud
 	Projectile: Bullet
-		Speed: 1c682
 		Blockable: false
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
 		Damage: 40
-		Versus:
-			Wood: 50
-			Light: 60
-			Heavy: 25
-			Concrete: 25
-		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
-	Warhead@2Eff: CreateEffect
-		Explosions: piffs
-		InvalidImpactTypes: Water
-	Warhead@3EffWater: CreateEffect
-		Explosions: water_piffs
-		ValidImpactTypes: Water
 
 Pistol:
+	Inherits: ^LightMG
```

Comment:
Due to the extremely low base damage, all except 'None' should get rounded down to zero damage anyway, so I chose saving some yaml lines instead.